var/spack/repos/builtin/packages/grabix/package.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+# @author:>-----Robert QIAO (DRS, Flinders University, Australia)
```

Comment:
I'm not sure if this line is worth adding since you are listed under `maintainers`

Reply:
Legacy header fixed.